### ISSUE TITLE
Supply custom metric to the neighbor embedding?

### ISSUE THREAD

**Issue by dhemgren** [jsilter/parametric_tsne#7]
Is there a way to supply a custom distance metric other than euclidian to the neighbor embedding? This would be very useful for feature vectors of varying sizes.

**Comment by jsilter** [comment]
There's nothing built-in to the API. You can supply your own beta coefficients pretty easily but that's it. In the high-dimensional space the distances are calculated by `get_squared_cross_diff_np`, for the low-dim space it takes a neural tensorflow neural network output and calculates using `_get_squared_cross_diff_tf`, any other distance metric would need to replace those.
